mithril-common/src/test_utils/mod.rs

```diff
@@ -17,7 +17,7 @@ mod cardano_transactions_builder;
 mod certificate_chain_builder;
 mod fixture_builder;
 mod mithril_fixture;
-
+mod precomputed_keskey;
```

Comment:
A better name for the module would probably be `precomputed_kes_key`

Reply:
Done